tests/test_migration.py

```diff
@@ -25,9 +26,93 @@ def test_migration(
     assert pytest.approx(strategy.estimatedTotalAssets(), rel=RELATIVE_APPROX) == amount
 
     # migrate to a new strategy
-    new_strategy = strategist.deploy(Strategy, vault)
+    name = "NewStrat"
+    new_strategy = strategist.deploy(
+        Strategy, vault, bento_box, kashi_pairs, pids, name
+    )
     vault.migrateStrategy(strategy, new_strategy, {"from": gov})
     assert (
         pytest.approx(new_strategy.estimatedTotalAssets(), rel=RELATIVE_APPROX)
         == amount
     )
+    assert new_strategy.name() == name
```

Comment:
can we check balances are correctly accurate on a migration? add some asserts around amounts or is this covered somewhere else?

Reply:
added